topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -27,7 +27,8 @@ It is a successor of _[](tools_gradle_intellij_plugin.md)_.
 
 ## Requirements
 
-IntelliJ Platform Gradle Plugin 2.x requires the following minimal versions:
+IntelliJ Platform Gradle Plugin 2.x requires the following **minimal versions**:
```

Comment:
I made the minimal in bold, but I think it will be better to write the '+' or "and above" next to each requirement